tests/test_helper/lxd.bash

```diff
@@ -0,0 +1,40 @@
+LXC_TEST_IMAGE=jammy
+DEPENDENCIES="jq"
+
+function start_containers() {
+    LXC_COUNT=$1
+    LXC_BASE_NAME="microovn-test"
+    ALL_CONTAINERS=""
+    for (( i = 0; i < LXC_COUNT; i++ )); do
+        CONTAINER_NAME="$LXC_BASE_NAME-$i"
```

Comment:
If we run multiple tests in parallel, they would most likely step on each other with this naming.

Does bats provide some variable identifying each test that we could use?